draft-ietf-quic-multipath.md

```diff
@@ -382,55 +382,50 @@ Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
 
 Both endpoints, namely the client and the server, can close a path,
 by sending PATH_ABANDON frame (see {{path-abandon-frame}}) which
-abandons the path with a corresponding Path Identifier. Once a path is
-marked as "abandoned", it means that the resources related to the path,
-such as the used connection IDs, can be released.
-However, information related to data delivered over that path SHOULD
-not be released immediately as acknowledgments can still be received
-or other frames that also may trigger retransmission of data on another
-path.
+abandons the path with a corresponding Path Identifier.
 
-The endpoint sending the PATH_ABANDON frame SHOULD consider a path as
+The endpoint sending the PATH_ABANDON frame can consider a path as
 abandoned when the packet that contained the PATH_ABANDON frame is
-acknowledged. When releasing resources of a path, the endpoint SHOULD
-send a RETIRE_CONNECTION_ID frame for the connection IDs used on the path,
-if any.
-
-The receiver of a PATH_ABANDON frame SHOULD NOT release its resources
-immediately, but SHOULD wait for the reception of the RETIRE_CONNECTION_ID
-frame for the used connection IDs or 3 RTOs.
+acknowledged.
+
+The receiver of a PATH_ABANDON frame should not release its resources
+immediately, but SHOULD wait for at least three times the current
+Probe Timeout (PTO) interval as defined in {{Section 6.2. of QUIC-RECOVERY}}
+after the last sent packet before sending the RETIRE_CONNECTION_ID frame
+for the corresponding CID.
+This is inline with the requirement of {{Section 10.2 of QUIC-TRANSPORT}}
+to ensure that paths close cleanly and that delayed or reordered packets
+are properly discarded.
+The effect of receiving a RETIRE_CONNECTION_ID frame is specified in the
+next section.
 
 Usually, it is expected that the PATH_ABANDON frame is used by the client
 to indicate to the server that path conditions have changed such that
 the path is or will be not usable anymore, e.g. in case of a mobility
-event. The PATH_ABANDON frame therefore indicates to the receiving peer
-that the sender does not intend to send any packets on that path anymore
-but also recommends to the receiver that no packets should be sent in
-either direction. The receiver of an PATH_ABANDON frame MAY also send
-an PATH_ABANDON frame to signal its own willingness to not send
+event. The PATH_ABANDON frame therefore recommends to the receiver
+that no packets should be sent on that path anymore.
+In addition, the RETIRE_CONNECTION_ID frame is used indicate to the receiving peer
+that the sender does not intend to send any packets assiciated to the
```

Comment:
The RETIRE_CONNECTION_ID frame is not an indication, it is stronger.
```suggestion
that the sender will not send any packets associated to the
```

Reply:
agreed